bundle-size/app.js

```diff
@@ -264,7 +290,8 @@ module.exports = app => {
         completed_at: new Date().toISOString(),
         output: {
           title: `${approvalMessagePrefix} | approved by @${approver}`,
-          summary: `${approvalMessagePrefix} | approved by @${approver}`,
+          summary: 'The bundle size (gzipped compressed size of `v0.js`) ' +
+            `of this pull request was specially approved by ${approver}`,
```

Comment:
I think you can remove the word "specially".

Reply:
Done.